sdk/schemaregistry/Azure.Data.SchemaRegistry/src/Serialization/SchemaRegistrySerializerOptions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Serialization;
+
+namespace Azure.Data.SchemaRegistry.Serialization
+{
+    /// <summary>
+    /// The options to use when configuring the <see cref="SchemaRegistrySerializer"/>.
+    /// </summary>
+    public class SchemaRegistrySerializerOptions
+    {
+        private static ObjectSerializer s_jsonObjectSerializer = new JsonObjectSerializer();
+
+        /// <summary>
+        /// Allows the user to pass in an <see cref="ObjectSerializer"/> with configured options.
+        /// The default is a <see cref="JsonObjectSerializer"/>.
+        /// </summary>
+        public ObjectSerializer Serializer { get; set; } = s_jsonObjectSerializer;
+
+        /// <summary>
+        /// The format of the schema to use for serialization. The default is JSON.
+        /// </summary>
+        public SchemaFormat Format { get; set; } = SchemaFormat.Json;
```

Comment:
Should we have validation that makes sure that if users change the Format, they also supply a custom serializer? e.g. it would be incorrect to set this to Avro and leave the Serializer as the default.

Reply:
Yeah this is a good idea - will update